content/broker/broker-transition/_index.en.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Altinn 3 Broker Transition Solution
+linktitle: Transition Solution
+description: Altinn 2 Broker Transition to Altinn 3 Broker architecture. 
+tags: [broker, transition, architecture]
+toc: false
+weight: 60
+---
+
+## About
+For broker service owners whose end users are reliant on using Altinn 2, but who wish to migrate their broker service to Altinn 3, Altinn is offering a Transition service.
+This enables communication between senders and receivers of files regardless of which Altinn Broker service they are integrated to.
+
+This service will allow service owners to link an existing Altinn 2 service to an Altinn 3 resource, and to forward Altinn 2 Broker Service requests to the Altinn 3 Broker Service app while not substantially changing the Altinn 2 Broker Service End User experience.
+This will then allow end users of the Altinn 2 service to communicate with end users of the Altinn 3 resource. 
+
+For Altinn 3 users, the user experience will not be different than if all users are using Altinn 3. 
+
+Altinn 2 users will experience some minor differences related to file transfer and returned values compared to if all users are using Altinn 2.
```

Comment:
Consider if "About" should be its own page to mirror the overall Ditaxis structure used for Broker.

Reply:
Har laget egne about sider og oppdatert liste. Tok bort Todo.